core/src/i18n/en.yaml

```
@@ -1240,6 +1247,7 @@ secrets:
       success: Secret created.
     title: Create Secret
   data: Data
+  description: <0>Secret</0> is an object that allows for storing sensitive data in key-value pairs.
```

I don't like the verb "allow" except when it's literally about permissions (so, for Oauth it made sense to me). Usually it actually means something along the lines of "enables", a possibility, not permission. Maybe we don't need it at all here?

```suggestion
  description: <0>Secret</0> is an object that stores sensitive data in key-value pairs.
```